openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -104,10 +149,19 @@ def report_completion(
 
     # Collect data into a single message
     message = f"""
-*Provider*: `{provider_name}`
+*DAG*: `{dag_id}`
 *Date range*: {date_range}
 *Duration of data pull task*: {duration or '_No data_'}
 *Number of records upserted per media type*:
 {media_type_reports}"""
+
+    if aggregate_duration:
+        # Add disclaimer about duration for aggregate data
+        message += (
+            "\n_Duration is the sum of the duration for each data pull task."
+            " It does not include loading time and does not account for data"
+            " pulls that may happen concurrently."
```

Comment:
I like this idea! Probably overkill but it would be interesting to have some kind of summary of outliers. For Wikimedia for instance the current large-batch issue could make for some wild outliers 😮

Reply:
Oh definitely 😰 Yea not necessary now but a potential option down the line if we find we want more distinction!